packages/@cdktf/provider-generator/lib/get/generator/models/attribute-model.ts

```diff
@@ -63,6 +63,11 @@ export class AttributeModel {
   }
 
   public get isAssignable() {
+    if (this.computed) {
+      // only computed && optional attributes are assignable
+      // if a computed attribute is required, it is never assignable
```

Comment:
Are you sure? That doesn't really make sense.

Reply:
That was the case we encountered with the snapshot tests – but yeah, it seems weird to have something computed & required.

Reply:
Which snapshot? I just tested the provider-generator tests without this changes and didn't notice anything.
I just want to make sure we aren't masking a different issue.

Reply:
It could have been the edge provider in which case we could've produced this weirdness ourselves 🤔 @DanielMSchmidt @skorfmann do you remember whether it was the edge provider? If that is the case, this change might not be needed and we should check the edge provider test case 😅

Reply:
I am unsure, I'd be down to removing this. If we run into issues we will probably see them arise soon